pkgs/development/python-modules/video2midi/default.nix

```diff
@@ -1,24 +1,24 @@
-{ stdenv, fetchFromGitHub, python2Packages, opencv3 }:
+{ stdenv, fetchFromGitHub, pythonPackages, opencv3 }:
 
 let
-  opencv3_ = python2Packages.toPythonModule (opencv3.override {
+  opencv3_ = pythonPackages.toPythonModule (opencv3.override {
     enablePython = true;
-    pythonPackages = python2Packages;
+    pythonPackages = pythonPackages;
     enableFfmpeg = true;
   });
 in stdenv.mkDerivation rec {
   pname = "video2midi";
-  version = "0.3.7";
+  version = "0.3.9.1";
 
   src = fetchFromGitHub {
     owner = "svsdval";
     repo = pname;
     rev = version;
-    sha256 = "00ms9iqam3ml6fxf8djki3gyifn8sfzfkkfrdgrvs0sq47wrkc39";
+    sha256 = "1ndzhfng8z5080n1xkcavw21dm6rjz0x1954v9llifsdmf4cpn8y";
   };
 
-  pythonPath = with python2Packages; [ opencv3_ midiutil pygame pyopengl ];
-  nativeBuildInputs = with python2Packages; [ python wrapPython ];
+  pythonPath = with pythonPackages; [ opencv3_ midiutil pygame pyopengl ];
```

Comment:
`pythonPath`  is deprecated and shouldn't be used for python applications.

Please use `propagatedBuildInputs`.